src/api/procedures/__tests__/inviteAccount.ts

```diff
@@ -109,16 +109,20 @@ describe('inviteAccount procedure', () => {
     dsMockUtils.configureMocks({
       contextOptions: {
         sentAuthorizations,
+        secondaryKeys: [
+          {
+            signer,
+            permissions: {
+              tokens: [],
+              portfolios: [],
+              transactions: [],
+              transactionGroups: [],
+            },
+          },
+        ],
```

Comment:
Is this still necessary? Shouldn't this option be removed from `ContextOptions` entirely?

Reply:
Ignore this